probes/hasDangerousWorkflowScriptInjection/def.yml

```diff
@@ -30,6 +30,11 @@ remediation:
   markdown:
     - Avoid the dangerous workflow patterns.
     - See [this document](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections) for information on avoiding and mitigating the risk of script injections.
+    - |
+      Here is a proposed patch to eliminate this risk:
+      ```yml
+      ${{ metadata.patch }}
+      ```
```

Comment:
Ack. We can always change it later if needed. 

P.S. The SARIF spec supports a `fix` object where this would be great for. But I'm unsure if GitHub supports it.

https://docs.oasis-open.org/sarif/sarif/v2.1.0/csprd01/sarif-v2.1.0-csprd01.html#_Toc10541319

Reply:
It does not, at least according to https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning#supported-sarif-output-file-properties.

Reply:
You would need to dig down further, as the `fix` object lives under the results object, which is supported. 
https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning#result-object

But yes, it appears not to be supported.